docs/blueprints/EP000.rst

```diff
@@ -0,0 +1,153 @@
+
+    - The file must be written in restructured text (RST) format like the other blueprints (as you can see at 'kytos/docs/blueprints').
+
+
+Blueprint review
+================
+
+In the following days after you submit your pull request, the Kytos team will review the document adding comments and suggestions. So the author of the pull request must pay attention to the Kytos team feedback to make the review a quick process.
+
+
+Sections that a blueprint should have
+*************************************
+
+Header
+======
+This section must use the template available in the Blueprint Header section of this blueprint.
+
+Abstract
+========
+A short (~200 word) description of the technical issue being addressed.
+
+Motivation
+==========
+It should clearly explain why the blueprint is being proposed describing any technical issue that is intended to be covered by the proposal.
+
+Rationale
+=========
+The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other environments or scenarios.
+
+Specification
+=============
+The technical specification should describe the aspects of what is being proposed.
+
+Backwards Compatibility
+=======================
+All EPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EP must explain how the author proposes to deal with these incompatibilities.
+
+Security Implications
+=====================
+If there are security concerns in relation to the EP, those concerns should be explicitly written out to make sure reviewers of the EP are aware of them.
+
+Rejected Ideas
+==============
+Throughout the discussion of an EP, various proposed ideas end up not being accepted. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This both helps record the thought process behind the final version of the EP as well as preventing people from bringing up the same rejected ideas again in subsequent discussions.
+ 
+References
+==========
+References -- A collection of URLs used as references through the EP.
+
+Copyright/license
+=================
+
+Types of blueprints
+*******************
+
+Standard: Describes a new feature and its implementation.
+
+Informational: Guidelines or information for the community and developers, but does not propose a new feature.
+
+Epic: Describes a problem and its solution.  
+
+
+################
+Blueprint Header
+################
+
+Every blueprint proposal should use this template to build their own header:
+
+:EP: <\*EP number>
+:Title: <EP title>
+:Authors: <list of authors' names and email addrs>
+:Created: <date created on, in yyyy-mm-dd format>
+:Kytos-Version: <kytos version, in yyyy.v>
+:Status: <Draft | Active | Accepted | Provisional | Deferred | Rejected | Withdrawn | Final | Superseded>
```

Comment:
I'm missing here, talk about all status possible. So we can evaluate if we really need all those statuses.

Reply:
I was discussing with Humberto and we agree that the following status are enough: Draft, Accepted, Rejected, Final and Superseded. Do you think that is necessary to explain each one?

Reply:
Yes, we need to explain  each state.